applicationset/utils/testdata/github-ping-event.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
Is this file used anywhere?

Reply:
It's being used in the applicationset controller's tests:https://github.com/argoproj/argo-cd/blob/afd50954ef26bb6b235a93be8bb8141a97e27394/applicationset/utils/webhook_test.go#L56

It's the same test data as the one I added in argocd's webhook tests (argocd and app-controller each have their own respective webhook testdata it seems). Did you want the application-set controller's webhook tests to use the test data in argocd's webhook tests folder? Or do you want them to remain separate?

Reply:
Nah, this works! I just didn't see the reference.